src/sql/workbench/browser/modelComponents/splitviewContainer.component.ts

```diff
@@ -48,7 +47,7 @@ class SplitPane implements IView {
 	`
 })
 
-export default class SplitViewContainer extends ContainerBase<FlexItemLayout> implements IComponent, OnDestroy {
+export default class SplitViewContainerImpl extends ContainerBase<FlexItemLayout> implements IComponent, OnDestroy {
```

Comment:
the class name has been changed, is this class not being used? I only see one occurrence of the new class name.

Reply:
I had to change the name since it clashes with `SplitViewContainer` from `azdata`

Reply:
The usage of this comes through the api, `modelBuilder.splitViewContainer()`

Reply:
https://github.com/microsoft/azuredatastudio/blob/8311c3985dc64721f15b8586e3cfa82a6a8daca9/src/sql/workbench/contrib/modelView/browser/components.contribution.ts#L44

you need to update this place

Reply:
No I don't, the export is a default export. It links back to the Impl

Reply:
Please check once that schema compare continues to work (as the splitviewcontainer is used by the same). Let us know if you need any help with the validation

Reply:
I'm not familiar with the end to end experience of Schema Compare, would it be possible for someone to check out this branch and test it locally?

Reply:
![image](https://user-images.githubusercontent.com/1465785/92532607-7cb0fc80-f1e5-11ea-8535-02aefcad0aa6.png)

Is this basically the splitView you mention @udeeshagautam?

Reply:
yup. I also checked out your branch and verified that the split view in schema compare still works as expected

Reply:
thanks!